service/storage_proxy.cc

```diff
@@ -6931,7 +6931,7 @@ future<> storage_proxy::abort_view_writes() {
 
 future<>
 storage_proxy::stop() {
-    return make_ready_future<>();
+    return await_pending_writes();
```

Comment:
Without aborting long running writes (currently MV timeout is 5 minutes or so) this may hang shutdown for a long time. For cleanup fiber this is less of a problem.

Reply:
I agree we should abort pending writes rather than wait for their timeout
but that said, I don't see we abort writes in table::stop(), so this is a pre-existing issue,
so I don't think it should block this PR.

Reply:
Do we support MV for tablets already?

Reply:
> Do we support MV for tablets already?

No, not in 2025.1.0.
But how is this specific to tablets?

Reply:
It is specific to MVs. Regular writes have timeout of 2 seconds. MVs have timeouts of 5 minutes IIRC.